tests/core/numerics/interpolator/test_main.cpp

```diff
@@ -541,6 +544,7 @@ class ACollectionOfParticles_1d : public ::testing::Test
             part.iCell[0] = 20; // AMR index
             part.delta[0] = 0.5;
             part.weight   = 0.4;
+            part.charge   = 1.85;
```

Comment:
While we (you) still remember, I think we should take 5minutes to add a little explanation for these numbers. Because I know that in a year we will not remember what they mean and why we chose these specific values.